ffi/ffi_wrapper.go

```diff
@@ -147,6 +150,8 @@ func TwinDelegatesControlToAgent(cResolverAddress *C.char,
 
 	cDelegationName *C.char) *C.char {
 
+	ctx := context.Background()
```

Comment:
`context.TODO` is [probably more appropriate](https://pkg.go.dev/context#TODO) - generally there is one BG context.

Reply:
[3e9b1c7](https://github.com/Iotic-Labs/iotics-identity-go/pull/46/commits/3e9b1c7c856ef721a0993e0fae4f7ac28f2398c1)